<launch>
  <group ns="bender">
    <rosparam file="$(find bender_arm)/config/rgbd_head_controller.yaml" command="load"/>
    <node name="head_controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
            args="--manager=r_arm_manager
                  --port=r_arm_port
                  rgbd_head_pitch_controller"
            output="screen"/>

    <rosparam file="$(find bender_arm)/config/rgbd_head_service.yaml" command="load"/>

    <node name="head_controller_spawner_meta" pkg="dynamixel_controllers" type="controller_spawner.py"
            args="--manager=r_arm_manager
                  --type=meta
                  rgbd_head_controller
                  rgbd_head_pitch_controller"
            output="screen"/>
  </group>
</launch>